README.md

```diff
@@ -1 +1,131 @@
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Event-Website] <a name="about-project"></a>
+
+
+**[Event-Website]** It is a website that helps different people around the world to make public their events on the web.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+## Html
+## Css
+## JavaScript
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[Introducing to the Event Website on the home page]**
+- **[Add more descripton of the website in the about page]**
+- **[Make the project responssive ]**
+- **[Interactive hamburger menu]**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://aubinleyoung.github.io/Event-Website/)
+- [Video description link](https://www.loom.com/share/87f765beddc2458b81db9225d9315f58)
+- [Project requirements Link](Not yet available)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+To get a local copy up and running follow these simple example steps.
+
+1. Download the zip file or clone with this command `git clone git@github.com:aubinleyoung/Event-Website.git`(Make sure you have `Git installed` in your Computer or Install it by following [these steps](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git))
+2. `cd Event-Website`
+3. Open it with the live server
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+👤 **Aubin SIMPEZE**
+
+- GitHub: [@githubhandle](https://github.com/aubinleyoung)
+- Twitter: [@twitterhandle](https://twitter.com/SimpezeAubin)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/aubin-simpeze-7a5b7a220/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Desktop and Mobile version]**
+- [ ] **[Add more animations]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/aubinleyoung/Event-Website/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my coding partners to support me when I got stuck and my code reviewers for the advice.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] You link is broken. Kindly fix this link to make your readme file descriptive and professional 😊